kpi/management/commands/remove_duplicate_assetversions.py

```diff
@@ -61,7 +61,7 @@ def find_original_and_duplicate_versions(version_pks, asset_pk):
             serialized = json.dumps((
                 version.deployed_content,
                 version.name,
-                version._deployment_data,
+                version._deployment_data, # noqa
```

Comment:
flagging these "improperly" spaced comments is my revenge for spending hours reviewing code that removes a leading underscore :fire:

Reply:
Nothing to say for my own defense :-D